.github/workflows/main.yaml

```diff
@@ -87,5 +87,6 @@ jobs:
         username: ${{ github.actor }}
         password: ${{ secrets.GITHUB_TOKEN }}
     - run: tar -xvf ./src.tar.gz
+    - run: make update-kopia-image
```

Comment:
Is this going to build an image on each Kanister commit into master?

Reply:
It would build but since the sah256 and tag are same, image won't be updated.

Reply:
But do we build and push a new image with unchanged binary every time?

Reply:
Yes, for now I haven't handled such scenario to check for existing image with same commit.

Reply:
As discussed offline, as a follow-up, we should make this a manual workflow. This should be updated only when we bump the commit ID.